adit/core/tests/test_tasks.py

```
@@ -155,3 +155,59 @@ def process(self):
+        status=DicomTask.Status.PENDING,
+        job=dicom_job,
+    )
+
+    dicom_job.queue_pending_tasks()
+
+    dicom_task.refresh_from_db()
+    assert dicom_task.queued_job is not None
+
+    error_message = "Connection refused by server"
+
+    def process(self):
```

Ruff (0.14.13)

Unused function argument: `self`

(ARG001)